### DISCUSSION TITLE
shrink or scale option

### DISCUSSION THREAD

**Discussion by Mario-vminion** [OrcaSlicer/OrcaSlicer#423] [Ideas]
hi will there be an option to scale or shrink the print ...no i want to use the x-y compensate but when i go further than0.2 there will be a strange blue line and when i put it to 0.15 there is non.. maybe some one can explain.![IMG_4488](https://user-images.githubusercontent.com/100128032/222903898-c2bb10b1-72ad-4ca1-a80f-de5d7b02b22a.jpg)

**Comment by bbinnard** [comment]
I too miss the Scale function - sometimes I like to scale an STL up or down and I can't find a way to do this.

**Comment by SoftFever** [comment]
@Mario-vminion @bbinnard 
The scale button is on the top assuming you are talking about scaling the model
![Screenshot 2023-05-27 at 7 11 39 PM](https://github.com/SoftFever/OrcaSlicer/assets/103989404/4bda3cde-09b7-4050-bff7-edd7fbba9807)

**Comment by bbinnard** [comment]
Yes, for some reason I missed seeing it. Thanks.
